internal/analysis/ppr-spike-findings.md

```
@@ -0,0 +1,736 @@
+): Promise<{ prelude: Readable }>
+```
+
+The Flight prerender generates a complete RSC payload for all resolved server
+components. It is distinct from the Fizz prerender -- it produces an RSC wire-
+format stream (not HTML). This Flight payload is consumed by
+`createFromNodeStream` to build the React element tree that Fizz then renders
+to HTML.
+
+### Layer Diagram
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)